Testing/Tests/graveyard_removal_test.py

```diff
@@ -0,0 +1,57 @@
+"""
+This class ensures that GraveyardRemoval.py correctly removes the graveyard from an h5m file.
+"""
+
+import os
+import pymoab
+import sys
+
+sys.path.insert(1,"../../PythonTool")
```

Comment:
Do you know if there is a better way to do that? looks "hacky".

Reply:
I think the solution is 
1. add the necessary files to make `PythonTool` a module
2. call pytest from the top level directory with the directory name of the tests as an argument.  

Then you can import things from `PythonTool` because it will be in the local directory from where you invoked pytest.  (@ypark234 does this in a different project)